Seqs_processing_tool/bioutils/module_3_input_output.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+
+def fastq_transfom_dict(path: str) -> dict:
+    counter = 4
```

Comment:
It is not really clear what counter does in function. Discription comment would be welcome.